WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -1,35 +1,328 @@
 import UIKit
 
-protocol LoginViewController {
-    func setupNavBarIcon()
-}
+/// This is the first screen following the log in prologue screen if the user chooses to log in.
+///
+class LoginEmailViewController: NUXAbstractViewController, SigninKeyboardResponder, LoginViewController {
+    @IBOutlet var instructionLabel: UILabel!
```

Comment:
I know how `IBOutlets` work, I just prefer not implicitly unwrapping them (despite Apple's guidance) since they can still cause unexpected crashes. Regardless, I'll try to stir up some team discussion—I thought we had such a discussion, but can't find reference to it so I'm probably thinking of something else…

Anyway, proceed 😀